### PR TITLE
[Data Hub] Add bioRxiv and medRxiv API data configurations

### PR DESCRIPTION
**Pull request by HazalCiplak**
https://github.com/elifesciences/data-hub-issues/issues/557
~Addition to that : Removed initial load configurations for biorxiv MECA path~

### CODE REVIEW

File: kustomizations/apps/data-hub-configs/web-api-data-pipeline.config.yaml [hunk 1]
```diff
@@ -280,32 +280,53 @@ webApi:
     table: biorxiv_medrxiv_meca_path_metadata
     dataUrl:
       urlExcludingConfigurableParameters: https://api.biorxiv.org/meca_index/elife/all
+
+  # bioRxiv/medRxiv apis
+  # bioRxiv api
   - dataset: '{ENV}'
-    table: biorxiv_medrxiv_meca_path_metadata
```
Comment (de-code) [inline kustomizations/apps/data-hub-configs/web-api-data-pipeline.config.yaml:284]: would maybe otherwise keep those changes in a separate PR (The commit history then provides an accurate history and it is easier to review)

**Review by de-code — DISMISSED** [review]

**Review by scottaubrey — APPROVED** [review]
